Sources/StripeKit/Core Resources/PaymentIntents/PaymentIntent.swift

```diff
@@ -229,9 +229,7 @@ public enum PaymentIntentCancellationReason: String, Codable {
     case duplicate
     case fraudulent
     case requestedByCustomer = "requested_by_customer"
-    case failedInvoice = "failed_invoice"
```

Comment:
@Patrick-Kladek Can you add back the other 3 cases? While we may not pass these to stripes explicitly, these types still need to be decoded from a response model.